node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/NodeSerializer.java

```diff
@@ -123,6 +124,7 @@ private void toSlime(Node node, Cursor object) {
         node.status().osVersion().ifPresent(version -> object.setString(osVersionKey, version.toString()));
         node.status().firmwareVerifiedAt().ifPresent(instant -> object.setLong(firmwareCheckKey, instant.toEpochMilli()));
         node.reports().toSlime(object, reportsKey);
+        node.modelId().ifPresent(modelId -> object.setString(modelIdKey, modelId));
```

Comment:
Please update `SerializationTest` to cover (de)serialization of this particular field.